common/src/main/java/org/jboss/hal/testsuite/cli/DomainManager.java

```diff
@@ -472,4 +475,105 @@ public boolean waitUntilAvailable(String host, long timeout) {
         }
         return isRunning;
     }
+
+    @Override
+    public void reloadIfRequiredAndWaitUntilRunning(long timeout) {
```

Comment:
To be honest I did not come with anything better and I think that after adding `Servers` word in method name would make it too long. Is not the javadoc sufficient in this case? Or what name would you suggest?